src/Slacklinker/Handler/Webhook.hs

```diff
@@ -153,11 +155,34 @@ handleMessage ev teamId = do
         userId <- recordUser workspaceId ev.user
         recordLink workspaceId userId linkSource linkDestination
 
+parseBotMessageAsUserMessage :: Value -> Maybe MessageEvent
+parseBotMessageAsUserMessage v = case fromJSON (copyField "app_id" "user" v) of
+  Success result -> Just result
+  Error _        -> Nothing
+  where 
+    -- FIXME: add a BotMessageEvent type to the slack-web library
```

Comment:
I'll do this soonish, but i dont have a dev environment set up for slack-web atm, so i figured i'd essentially hack this together so we can have reacji channeler backlinks asap